README.md

```diff
@@ -15,137 +15,146 @@ OR
 
 ## App Configuration
 
-#### Android: 
+#### Android:
 
 1. Modify `android/app/build.gradle`:
 
-    1.1. Add to `android` section:
+   1.1. Add to `android` section:
 
-    ```
-    lintOptions {
-        abortOnError false
-    }
-    ```
+   ```
+   lintOptions {
+       abortOnError false
+   }
+   ```
+
+   1.2 Add to `android` -> `defaultConfig` section:
 
-    1.2 Add to `android` -> `defaultConfig` section:
-    
-    ```
-    minSdkVersion 19
-    multiDexEnabled true
-    ```
+   ```
+   minSdkVersion 23
+   multiDexEnabled true
+   ```
 
-    1.3 Add to `dependencies` section:
+   1.3 Add to `dependencies` section:
 
-    ```
-    implementation 'androidx.multidex:multidex:2.0.1'
-    ```
+   ```
+   implementation 'androidx.multidex:multidex:2.0.1'
+   ```
 
 2. Add to `android/app/proguard-rules.pro`:
-    ```
-    -keep class com.onegini.mobile.SecurityController { *; }
-    ```
+
+   ```
+   -keep class com.onegini.mobile.SecurityController { *; }
+   ```
 
 3. Add to `android/build.gradle`[allprojects.repositories]:
 
-    ```
-    dependencies {
-            classpath("com.android.tools.build:gradle:4.1.1")
-            classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.10")
-        }
-
-    ```
-
-    ```
-    mavenCentral()
-    if (project.hasProperty('onegini_artifactory_user') && project.hasProperty('onegini_artifactory_password')) {
-        maven {
-            /*
-            Before the release please change the url below to: https://repo.onegini.com/artifactory/onegini-sdk
-            Please change it back to https://repo.onegini.com/artifactory/public after the release
-            */
-            url "https://repo.onegini.com/artifactory/onegini-sdk"
-            credentials {
-                username "${onegini_artifactory_user}"
-                password "${onegini_artifactory_password}"
-            }
-        }
-    } else {
-        throw new InvalidUserDataException("You must configure the 'onegini_artifactory_user' and 'onegini_artifactory_password' properties in your project before you can " +
-                "build it.")
-    }
-    ```
+   ```
+   dependencies {
+           classpath("com.android.tools.build:gradle:4.1.1")
+           classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.10")
+       }
```

Comment:
Also, the dependencies should be added to `buildscript` instead of `allprojects` and/or `allprojects.repositories`.